### PR TITLE
Updated R4-Location with new search parameters

### PR DESCRIPTION
**Pull request by Surya-Rnsit**
Description
----
updated R4-Location with new search parameters

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
Screenshot (82)
Screenshot (83)
![Screenshot (81)](https://user-images.githubusercontent.com/87635660/1361
Screenshot 2021-12-01 at 4 39 48 PM
Screenshot 2021-12-01 at 4 39 56 PM

### CODE REVIEW

File: content/millennium/r4/base/entities/location.md [hunk 1]
```diff
@@ -53,11 +53,18 @@ Search for Locations that meet supplied query parameters:
...
+----------------------|-------------------------|--------------|-------------------------------------------------------
+ `_id`                | This or `-physicalType` | [`token`]    | The logical resource id associated with the resource.
+ `-physicalType`      | This or `_id`           | [`token`]    | The location’s physical type. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
+ [`_count`]           | no                      | [`number`]   | The maximum number of results to return. Defaults to `100`.
+ `address`            | no                      | [`string`]   | A (part of the) address of the location.
+ `address-city`       | no                      | [`string`]   | A city specified in an address
+ `address-state`      | no                      | [`string`]   | A state specified in an address
+ `address-postalCode` | no                      | [`string`]   | A postal code specified in an address
+ `name`               | no                      | [`string`]   | A portion of the location's name or alias
+ `organization`       | no                      | [`reference`]| Searches for locations that are managed by the provided organization
```
Comment (NeethuKP) [inline content/millennium/r4/base/entities/location.md:66]: Can you include the examples for name and organization serach

**Review by vinuthnag — APPROVED** [review]

File: content/millennium/r4/base/entities/location.md (hunk 1 above)
Comment (Surya-Rnsit) [reply content/millennium/r4/base/entities/location.md:66]: added 5d54dc306a2722c759d56ff65ad300ddcc83078b

File: lib/resources/example_json/r4_examples_location.rb [hunk 2]
```diff
@@ -95,5 +95,244 @@ module Resources
...
+        ],
+        "city": 'Malvern',
+        "state": 'PA',
+        "postalCode": '19355',
+        "country": 'USA'
+      },
+      "physicalType": {
+        "coding": [
+          {
+            "system": 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/222',
```
Comment (vinuthnag) [inline lib/resources/example_json/r4_examples_location.rb:132]: change domain to ec2458f2-1e24-41c8-b71b-0e701af7583d

File: lib/resources/example_json/r4_examples_location.rb [hunk 3]
```diff
@@ -95,5 +95,244 @@ module Resources
...
+              ],
+              "city": 'Kansas City',
+              "state": 'MO',
+              "postalCode": '64117',
+              "country": 'US'
+            },
+            "physicalType": {
+              "coding": [
+                {
+                  "system": 'https://fhir.cerner.com/dacc6494-e336-45ad-8729-b789ff8663c6/codeSet/222',
```
Comment (vinuthnag) [inline lib/resources/example_json/r4_examples_location.rb:240]: change system domain to ec2458f2-1e24-41c8-b71b-0e701af7583d

File: lib/resources/example_json/r4_examples_location.rb [hunk 4]
```diff
@@ -95,5 +95,244 @@ module Resources
...
```
Comment (vinuthnag) [inline lib/resources/example_json/r4_examples_location.rb:315]: change system domain to ec2458f2-1e24-41c8-b71b-0e701af7583d

**Review by vinuthnag — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_location.rb (hunk 4 above)
Comment (Surya-Rnsit) [reply lib/resources/example_json/r4_examples_location.rb]: Addressed 9ff8663c6

File: lib/resources/example_json/r4_examples_location.rb (hunk 2 above)
Comment (Surya-Rnsit) [reply lib/resources/example_json/r4_examples_location.rb]: addressed https://github.com/cerner/fhir.cerner.com/pull/749/commits/4e79521abc896ef5a9ef28e79b25a6842781919b

File: lib/resources/example_json/r4_examples_location.rb (hunk 3 above)
Comment (Surya-Rnsit) [reply lib/resources/example_json/r4_examples_location.rb]: addressed https://github.com/cerner/fhir.cerner.com/pull/749/commits/4e79521abc896ef5a9ef28e79b25a6842781919b

**Review by vinuthnag — APPROVED** [review]

**Comment by aki-mathur** [comment]
Screenshot 2021-12-02 at 6 19 34 PM

**Pull request merged**